packages/api/models/granules.js

```diff
@@ -634,7 +634,7 @@ class Granule extends Manager {
    */
   _getMutableFieldNames(record) {
     if (record.status === 'running') {
-      return ['updatedAt', 'timestamp', 'status', 'execution'];
+      return ['createdAt', 'updatedAt', 'timestamp', 'status', 'execution'];
```

Comment:
*CreatedAt* should be mutable - as it's being explicitly set by the `workflow_start_time` in `cumulus_meta`, we *want* any  successful write to update this.

Reply:
You raise an excellent point, *very* good catch.  

It should.  In practice, it shouldn't cause a real precedence issue, but that's a happy accident.   I checked and the other models (e.g. executions) already do this.  

https://github.com/nasa/cumulus/pull/2133